analyses/oncoprint-landscape/README.md

```diff
@@ -1,30 +1,49 @@
-# OpenPBTA Oncoprint
+## OpenPBTA Oncoprint Landscape
 
-## Work in progress
+**Module authors:** Chante Bethell ([@cbethell](https://github.com/cbethell)) and Jaclyn Taroni ([@jaclyn-taroni](https://github.com/jaclyn-taroni))
 
-This module is a work in progress.
-The code was written to be updated as consensus, filtered, and/or prioritized data becomes available.
-**The plots included as PNGs should be regarded as proof of concept, rather than for interpretation.**
+The purpose of this module is to plot the landscape of the consensus CNV, SNV and fusion OpenPBTA datasets.
 
-In particular, the following are likely to change:
+### Usage
 
-* The plot will use consensus copy number calls ([#128](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/128)).
-
-## Usage
-
-To run the Rscript in this module from the command line as intended, use:
+To run the Rscripts in this module from the command line as intended, use:
 
 ```
 bash run-oncoprint.sh
 ```
 
 `run-oncoprint.sh` is designed to be run as if it was called from this module directory even when called from outside of this directory.
 
-## Folder content
+### Folder content
 
 * `00-map-to-sample_id.R` prepares MAF, focal CN (the output of the `focal-cn-file-preparation` module), and standardized fusion files for use with `01-plot-oncoprint.R`. 
   * The `Tumor_Sample_Barcode` column in the output corresponds to the `sample_id` column in the histologies file
   * We remove ambiguous `sample_id` -- i.e., where there are more than two tumor biospecimens that map to the same sample identifier.
   * Filtering via an [independent specimen file](https://alexslemonade.github.io/OpenPBTA-manuscript/#selection-of-independent-samples) is optional, but highly recommended.
-* `01-plot-oncoprint.R` takes the files from above and optionally a gene list and creates an oncoprint.
+* `01-plot-oncoprint.R` takes the files from above and optionally a list of top mutated genes (generated in `analyses/interaction-plots/scripts/01-disease-specimen-lists.R`, top genes with recurrent CNVs (generated in `analyses/focal-cn-file-preparation/06-find-recurrent-calls.Rmd`, or a concatenated list of these genes (generated in this module's `util/concatenate-gene-lists.R` script).
+This script generates an oncoprint displaying the landscape across PBTA given
+the relevant metadata.
+* `concatenate-gene-lists.R` takes two separate gene lists as input and combines them into a list with one column for use in the oncoprint plotting.
+The output of this script is a TSV file with the combined list of genes, stored in this module's `results` directory.
+This script is found in this module's `util` directory.
+
+### Folder Structure
```

Comment:
You added back the driver lists with https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/711/commits/cb210ea1fbd15c00d152dc2f305e9f676d2191f5. They should be represented here and also the source should be documented as stated in #706